system-addon/test/unit/unit-entry.js

```diff
@@ -0,0 +1,11 @@
+const req = require.context(".", true, /\.test\.js$/);
+const files = req.keys();
+
+// This exposes sinon assertions to chai.assert
+sinon.assert.expose(assert, {prefix: ""});
```

Comment:
Ah, indeed.  At some point, we might consider switching to https://github.com/domenic/sinon-chai for test readability.